static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -488,21 +514,24 @@ const currentBidders: (HeaderBiddingSize[]) => PrebidBidder[] = slotSizes => {
         ...(shouldIncludeOpenx() ? [openxClientSideBidder] : []),
     ];
 
-    const allBidders = indexExchangeBidders(slotSizes)
-        .concat(otherBidders);
+    const allBidders = indexExchangeBidders(slotSizes).concat(otherBidders);
     return isPbTestOn()
         ? biddersBeingTested(allBidders)
         : biddersSwitchedOn(allBidders);
 };
 
-export const bids: (string, HeaderBiddingSize[]) => PrebidBid[] = (
-    slotId,
-    slotSizes
-) =>
-    currentBidders(slotSizes).map((bidder: PrebidBidder) => ({
-        bidder: bidder.name,
-        params: bidder.bidParams(slotId, slotSizes),
-    }));
+export const bids: (
+    string,
+    HeaderBiddingSize[]
+) => Promise<PrebidBid[]> = async (slotId, slotSizes) => {
+    const currentBiddersAsync = await currentBidders(slotSizes);
+    return Promise.all(
+        currentBiddersAsync.map(async (bidder: PrebidBidder) => ({
+            bidder: bidder.name,
+            params: await bidder.bidParams(slotId, slotSizes),
+        }))
+    );
```

Comment:
This will fail if any promise fails. We could consider using [`Promise.allSettled()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled).